molgenis-dataexplorer/src/test/java/org/molgenis/dataexplorer/DataExplorerControllerTest.java

```diff
@@ -73,12 +73,12 @@ public void beforeTest()
 	public void getAnnotatorModuleSuccess() throws Exception
 	{
 		assertEquals("view-dataexplorer-mod-" + DataExplorerController.MOD_ANNOTATORS,
-				controller.getModule(DataExplorerController.MOD_ANNOTATORS, "yes", mock(Model.class)));
+				controller.getModule(DataExplorerController.MOD_ANNOTATORS, "yes", mock(Model.class), null));
```

Comment:
You fixed this existing test but you also should create a new one, to test the new situation.